cypher-shell/src/test/java/org/neo4j/shell/parser/ShellStatementParserTest.java

```diff
@@ -275,12 +275,12 @@ public void commentHandlingIfSemicolon() throws Exception {
 
         // then
         assertTrue(parser.hasStatements());
-        assertTrue(parser.containsText());
+        assertFalse(parser.containsText());
 
         List<String> statements = parser.consumeStatements();
 
         assertEquals(1, statements.size());
-        assertEquals(" first // ;\n// /* ;\n third ;", statements.get(0));
+        assertEquals(" first  third ;", statements.get(0));
```

Comment:
Why did you remove the comments from these tests here?

Reply:
we no longer pass along the comment strings to the server. This test sends

```
 first // ;
// /* ;
 third ; // actually a semicolon here
```
now we strip out everything that is actually a comment.